s2i/config/krb5.conf

```diff
@@ -1,30 +1,36 @@
-# To opt out of the system crypto-policies configuration of krb5, remove the
-# symlink at /etc/krb5.conf.d/crypto-policies which will not be recreated.
 includedir /etc/krb5.conf.d/
 
-[logging]
-    default = FILE:/var/log/krb5libs.log
-    kdc = FILE:/var/log/krb5kdc.log
-    admin_server = FILE:/var/log/kadmind.log
+# depending on your config, you may wish to uncomment the following:
+# includedir /var/lib/sss/pubconf/krb5.include.d/
 
 [libdefaults]
-    dns_lookup_realm = false
-    ticket_lifetime = 24h
-    renew_lifetime = 7d
-    forwardable = true
-    rdns = false
-    pkinit_anchors = FILE:/etc/pki/tls/certs/ca-bundle.crt
-    spake_preauth_groups = edwards25519
-    default_realm = REDHAT.COM
-    dns_canonicalize_hostname = fallback
+  default_realm = IPA.REDHAT.COM
+  dns_lookup_realm = true
+  dns_lookup_kdc = true
+  rdns = false
+  dns_canonicalize_hostname = false
+  ticket_lifetime = 24h
+  forwardable = true
+  udp_preference_limit = 0
+  default_ccache_name = KEYRING:persistent:%{uid}
```

Comment:
I would recommend leaving this to either default or setting it as a file in /tmp.

Using keyring could mean that other pods/containers on the same node can access the krb key. Plus I think in some cases the support might not be there?

Reply:
@sochotnicky I removed the ccaching setup, could you check please?